contents/blog/10x-engineers-do-user-interviews.md

```diff
@@ -0,0 +1,104 @@
+---
+date: 2023-02-19
+title: Becoming a 10x engineer - why user interviews are key
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - luke-harries
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+  - Guides
+  - Product engineers
+---
+
+When it comes to becoming a 10x engineer, most people think about improving their typing speed, buying more monitors, or using the latest framework where everything is a macro. But, in reality, leverage comes from choosing the right problem, building the right solution, and using the right tools. This can only happen when you have a deep understanding of your users. It's the only way to build something that people want.
+
+## Velocity vs. Speed
+
+```jsx
+Speed = how quickly you can go from an idea to a rolled-out implementation.
+
+Velocity = how quickly and how large an impact you can have on your users.
+```
+
+It's easy to fall into the trap of optimizing only for speed. But you might rush to implement a feature only to find no one uses it. That's why it's crucial to speak directly with your users. By understanding their pain points and needs, you can design solutions that are intuitive and effective, addressing their "hair-on-fire" problems. Moving quickly is important, but it's even more important to move quickly in the right direction.
```

Comment:
I feel like the "velocity vs speed" section bit here is a bit of a distraction and the "this article is about talking to users" gets a bit lost as result. Imo, it would be best write it as a two paragraph intro where the 1st para covers why optimising for speed is bad, and the 2nd para covers why 'talking to users' is the answer and what the reader will get out of this article – i.e. advise on how to talk to users as an engineer.

Reply:
Good point, will do!